capacity4more/behat/features/bootstrap/FeatureContext/Homepage.php

```diff
@@ -30,6 +30,77 @@ public function iShouldSeeTheSiteHomepage() {
   }
 
   /**
+   * @Then /^I should see the featured block$/
+   */
+  public function iShouldFeaturedBlock() {
+    $steps[] = new Step\When('I should see "Featured block"');
+    $steps[] = new Step\When('I should see "Nobel Foundation" in the "div.pane-featured-block" element');
+    $steps[] = new Step\When('I should see "Nobel Banquet" in the "div.pane-featured-block" element');
+    $steps[] = new Step\When('I should see "Second Prize" in the "div.pane-featured-block" element');
```

Comment:
@nader77 It looks not correct, because every time we visit the homepage - there will be 3 random featured items, not these three every time.